pkg/pillar/types/newlogtypes.go

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) 2020 Zededa, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package types
+
+import (
+	"time"
+)
+
+const (
+	// DevPrefix - device log file prefix string
+	DevPrefix = "dev.log."
+	// AppPrefix - app log file prefix string
+	AppPrefix = "app."
+	// AppSuffix - app log file suffix string, the appuuid is between the AppPrefix and AppSuffix
+	AppSuffix = ".log."
+)
+
+type logfileMetrics struct {
+	// from newlogd
+	NumGZipFilesSent  uint64 // total gzip files uploaded, device
+	NumGZipBytesWrite uint64 // total gzip log in bytes, device
+	NumBytesWrite     uint64 // total log bytes write to file before gzip, device
+	NumGzipFileInDir  uint32 // current number of gzip files remain, device
+	NumInputEvent     uint64 // total event input from log source, device include kmessages
```

Comment:
Does "device" here mean it excludes the app logs? Or something else?

Reply:
Correct. 'device' means not App log related.

Reply:
That appears to be incorrect, because you use this struct as 
	DevMetrics logfileMetrics // Device metrics
	AppMetrics logfileMetrics // App metrics

So you can drop the ", device" from the comments and maybe say before the logfileMetrics that it is used to separately track device and app metrics?

Reply:
makes sense, remove the 'device' in comments of the struct.